ci-operator/step-registry/openshift/e2e/cert-rotation/openshift-e2e-cert-rotation-workflow.yaml

```diff
@@ -0,0 +1,19 @@
+workflow:
+  as: openshift-e2e-cert-rotation
```

Comment:
The env `FEATURE_GATES: '["ShortCertRotation=true"]'` is set, if we use `openshift-e2e-short-cert-rotation` name here, the feature related to this test can be known more clearly.

Reply:
That would make us move out of `cert-rotation` folder and make it harder to setup custom OWNERS.

Its unfortunate though, lets use `openshift-e2e-cert-rotation-short` instead?

Reply:
lgtm